Server/src/models/VariableModel.ts

```diff
@@ -1,5 +1,5 @@
 import { Datapoints } from "./entities/Datapoints";
-import { IVariableNameModel } from "../../../Client/src/Models/IVariableNameModel"
+import { IVariableNameModel } from "../../../Client/src/Models/Variables/IVariableNameModel"
```

Comment:
It would be better to have a copy of this model inside of the Server folder instead of referencing a model from Client.

Reply:
I honestly don't know why I faced the issue about the reference there. I didn't check Dev but a merge from Dev didn't solve it. So I renamed the model import instead. Also referencing models between client and server has been done everywhere, it would be better to use it this way rather than having duplicate files in both folders.

Reply:
No they shouldnt cross reference, cross referencing the models like that couples cloent and server together, I also removed previous crossreferencing that was introduced with dimensions and variable names PRs

Reply:
It just happened that we have client and aerver code in one repo, but ideally they would be two separate repos, we shouldnt crossreference them like that. One more example is that server models can haveore data that the client doesnt care about

Reply:
OK removing it to server in the next push

Reply:
Thanks for this change, appreciated!